### PR TITLE
Stops Mine Drones from bleeding

### LINKED ISSUE #8643: Robotic critters (Mining rovers, etc) bleed when shot.

### PR DESCRIPTION
**Pull request by Baa14453**
Mine drones inherit the simple animal code instead of silicon so they bled blood and it was terrifying.
What did NanoTrasen put in there?!
Fixes #8643

### CODE REVIEW

**Comment by Baa14453** [comment]
!review

**Comment by Doxxmedearly** [comment]
This just cleans up some things matt missed in his bullet impact PR. It's exactly what happens when other synths are shot. Goofy paths meant a few got missed.

**Comment by Baa14453** [comment]
!bugfix

File: code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm [hunk 1]
```diff
@@ -173,4 +173,7 @@
 
 /mob/living/simple_animal/hostile/retaliate/minedrone/fall_impact()
 	visible_message("<span class='danger'>\The [src] bounces harmlessly on its inflated wheels.</span>")
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+mob/living/simple_animal/hostile/retaliate/minedrone/get_bullet_impact_effect_type(var/def_zone)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm:178]:
```suggestion
/mob/living/simple_animal/hostile/retaliate/minedrone/get_bullet_impact_effect_type(var/def_zone)
```

Comment (Baa14453) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm:178]: That's wild it even compiled
I must've just been really unlucky during testing

**Pull request merged**